test/unit_tests/test_ops.py

```diff
@@ -48,16 +50,24 @@
 @pytest.mark.parametrize("angle", [(i + 1) * math.pi / 12 for i in range(12)])
 def test_ops_parametrized(pl_op, braket_gate, angle):
     """Tests that the matrices and decompositions of parametrized custom operations are correct."""
-    assert np.allclose(pl_op._matrix(angle), braket_gate(angle).to_matrix())
+    assert np.allclose(pl_op.compute_matrix(angle), braket_gate(angle).to_matrix())
     _assert_decomposition(pl_op, params=[angle])
 
 
+@pytest.mark.parametrize("pl_op, braket_gate", gates_2q_parametrized)
+def test_ops_parametrized_tf(pl_op, braket_gate):
```

Comment:
Why not do the same test as above in test_ops_parametrized? just feed in a tf.Variable instead.